pkgs/utils/src/log.zig

```diff
@@ -100,3 +106,26 @@ pub fn getScopedLogger(comptime scope: LoggerScope, activeLevel: ?std.log.Level)
 pub fn getLogger(activeLevel: ?std.log.Level) ZeamLogger {
     return ZeamLogger.init(std.log.default_log_scope, activeLevel orelse std.log.default_level);
 }
+
+pub fn getFormattedTimestamp(buf: []u8) []const u8 {
+    if (builtin.target.os.tag == .freestanding) {
```

Comment:
you don't need to do this if now since we dont even call in freestanding/inside zkvm